common/src/main/java/org/valkyrienskies/mod/mixin/feature/entity_collision/MixinServerEntity.java

```diff
@@ -0,0 +1,91 @@
+package org.valkyrienskies.mod.mixin.feature.entity_collision;
```

Comment:
Instead of putting this in entity_collision, can you move this custom packet stuff to its own feature `custom_movement_packets`?

Reply:
Explained via comment.